homework/03LessonThree.js

```diff
@@ -20,18 +20,21 @@ var firstName = 'susan';
 
 // Problem 1:
 // What is the value of firstName at this point in the file?
-var valueOfFirstName;
+var valueOfFirstName = 'susan';
 
 // Problem 2:
 // What is the value of lastName at this point in the file?
-var valueOfLastName;
+var valueOfLastName = 'plaid';
 
 // Problem 3:
 // Fix the following function so that it returns the first argument
+var firstArgument = 'bob'
+
 function returnTheFirstArgument(firstArgument) {
   var firstArgument = 'bob';
   return firstArgument;
 }
+returnTheFirstArgument();
```

Comment:
mocha tests will run all functions automatically. No need to declare a function here..